src/coreclr/scripts/superpmi_collect_setup.py

```diff
@@ -383,28 +389,33 @@ def main(main_args):
     superpmi_src_directory = os.path.join(source_directory, 'src', 'coreclr', 'scripts')
     superpmi_dst_directory = os.path.join(correlation_payload_directory, "superpmi")
     arch = coreclr_args.arch
+    platform_name = coreclr_args.platform.lower()
     helix_source_prefix = "official"
     creator = ""
     ci = True
-    if is_windows:
+    if platform_name == "windows":
         helix_queue = "Windows.10.Arm64" if arch == "arm64" else "Windows.10.Amd64.X86.Rt"
-    else:
+    elif platform_name == "linux":
         if arch == "arm":
             helix_queue = "(Ubuntu.1804.Arm32)Ubuntu.1804.Armarch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm32v7-bfcd90a-20200121150440"
         elif arch == "arm64":
             helix_queue = "(Ubuntu.1804.Arm64)Ubuntu.1804.ArmArch@mcr.microsoft.com/dotnet-buildtools/prereqs:ubuntu-18.04-helix-arm64v8-20210531091519-97d8652"
         else:
             helix_queue = "Ubuntu.1804.Amd64"
+    elif platform_name == "osx":
+        helix_queue = "OSX.1100.ARM64" if arch == "arm64" else "OSX.1014.Amd64"
 
     # create superpmi directory
     print('Copying {} -> {}'.format(superpmi_src_directory, superpmi_dst_directory))
     copy_directory(superpmi_src_directory, superpmi_dst_directory, verbose_output=True, match_func=lambda path: any(path.endswith(extension) for extension in [".py"]))
 
-    if is_windows:
+    if platform_name == "windows":
```

Comment:
nit: This can continue to be `is_windows`?

Reply:
Well, it could. 
But `is_windows`, in fact, means `is_host_windows` while when preparing the artifacts to publish to the Helix we actually mean `is_target_windows`. I thought it would make more sense to update the condition to reflect our intention.

Reply:
Decided to leave this as is for the reason I described.